source/content/wordpress-cron.md

```diff
@@ -1,56 +1,108 @@
 ---
 title: Cron for WordPress
-description: Configuring and optimizing the WP-Cron feature on your Pantheon WordPress site.
+description: Learn how to create and run jobs using Cron or WordPress's WP-Cron feature on your Pantheon site.
 cms: "WordPress"
 categories: [automate]
 tags: [cron]
+contributors: [whitneymeredith]
 ---
 
+## Cron for WordPress Overview
+
+Cron is a standard utility in Unix and Linux systems that it is used to schedule commands for automatic execution at configured intervals. These scheduled commands or tasks are known as Cron Jobs. The name Cron comes from the Unix system for scheduling jobs, ranging from once a minute to once a year. Cron is generally used for running scheduled backups, monitoring disk space, deleting files that are no longer required, running system maintenance tasks, and much more. WP-Cron must be disabled to allow Cron to schedule and run tasks.
```

Comment:
There were already some comments in Slack regarding the terminology used in "WP-Cron must be disabled to allow Cron to schedule and run tasks" -- i.e. casually speaking, "WP-Cron" and "Cron" seem to be the same thing. This is, at least, true in the introduction.

It is also not completely accurate that WP-Cron must be disabled for Pantheon's periodic task to run the WordPress cron job. Now that Pantheon is calling WordPress cron every hour, though, there is a benefit in terms of efficiency to disabling WP-Cron, as once this is done, the WP-Cron job will no longer add load to page requests.